### PR TITLE
Fix  #294 Vesting Stats

### LINKED ISSUE #294: [1.5][happyconcepts] More intuitive vesting BTS stats
The stats shown under the "Membership" tab are confusing to me. I would expect "Pending vested fees" to match the amount of BTS I have vesting from my referee's use of the system; instead it is 0.

screen shot 2017-08-19 at 10 21 01 am

On the "Vesting balances" page, there are more accurate stats. Should the stats be removed from "Membership" entirely? Can we improve the intuition of these stats and unify them under one place? Perhaps there should be two tabs, one for worker pay stats and another for referral pay stats?

screen shot 2017-08-19 at 10 21 27 am

### PR DESCRIPTION
**Pull request by happyconcepts**
Fix #294 (english language).

### CODE REVIEW

File: app/assets/locales/locale-en.json [hunk 1]
```diff
@@ -136,9 +136,9 @@
       "network_percentage": "Network",
       "fee_allocation": "Fee Allocation",
       "membership": "Membership",
-      "fees_paid": "Lifetime fees paid",
-      "fees_pending": "Pending fees",
-      "fees_vested": "Pending vested fees",
+      "fees_paid": "Fees paid, Lifetime",
+      "fees_pending": "Fees pending division",
```
Comment (svk31) [inline app/assets/locales/locale-en.json]: Not sure what this means, pending what division?

**Comment by happyconcepts** [comment]
The description uses the word "divided" to describe the affiliate,
registrar, and network fee splits that occur.

 Whereas `Pending Vested Fees` are not divided at all.

**Comment by svk31** [comment]
I'm still confused as to what these two are and how they work. I just watched the openledger account register a new account, and that added to "Pending vested fees", even though the fee for that is only 1.2 BTS so it shouldn't need to vest :/

**Comment by happyconcepts** [comment]
@svk31 they should not have named that data point "vested" so you are seeing the expected behavior. I would presume you saw the value go back to 0 within the hour right?

**Comment by wmbutler** [comment]
Hoping the terminology I added will work.

**Comment by happyconcepts** [comment]
Kind of wish now i didn't spend 2 hours wrestling with this on the testnet but I took it seriously.

**Comment by wmbutler** [comment]
@happyconcepts I won't take the payment if it's accepted. It'll still credit to you. I do think the terminology is confusing and needs a solution. Don't lose hope.

**Comment by svk31** [comment]
In my understanding lifetime_fees_paid just means the total amount of fees paid by the account in its lifetime, nothing to do with lifetime membership.

Pending fees sounds like fees you haven't paid yet but will have to eventually.

**Comment by happyconcepts** [comment]
Gentlemen, internet access is situationally limited for me at the moment, precluding me from extended discussions.... but here is the most constructive thing I can mention in order to enlighten you:

1. There is a large number of transactions being processed and added to the chain at short intervals of time.
2. Think about what BM did. Or rather, had to do. To process the affiliate splits on transaction fees when he didn't have enough block time to work with.
3. So these two fields pending_vested_fees and pending_fees are created as temporary storage to track fees. Every hour the chain processes these amounts for fee splits and then these values are reset to zero. This is why most of the time these values are shown as 0 in the UI.
4. So `Pending` has been the source of confusion here because these temp fields are pending processing by the chain. They are not pending the completion of a vesting schedule which is how the word `pending` is used in other parts of the UI

My opinion is that you need the decide the tradeoff: On the one hand, this information is not necessary for the end user, and it obviously confusing. And on the other hand, this is a reference client, and this information is necessary for developers to understand how the affiliate logic is being implemented to the chain.

If it were me, I would choose to continue displaying these values but I would first de-emphasize them via CSS style and then augment the descriptive information so that the user knows what they are looking at. And I would choose to avoid the use of the word `pending` in order to avoid confusion with vesting schedules.

and yes, this bounty would help[ me alleviate my internet access situation so I am motivated for progress on this one.

**Comment by svk31** [comment]
Ok; so that explanation makes things a lot clearer. To be honest I don't see the value of even displaying these fields then, since they'll probably only be confusing to users. 

Also, fees_paid should simply be changed to "Total fees paid" since that's what it is. I will accept a PR with the above text change and removal of the two fees_pending and fees_vested fields.

Happy to let you have the bounty for that since you've clearly spent some time digging into this even if the final changes are minor.

**Comment by svk31** [comment]
Made the modifications myself and merged here: https://github.com/bitshares/bitshares-ui/pull/575

Like I said above, @happyconcepts should still get the bounty.

**Pull request closed without merging**